main.nf

```diff
@@ -646,6 +647,67 @@ def d4_coverage_version(task) {
 	"""
 }
 
+process verifybamid2 {
+        cpus 16
+        memory '10 GB'
+        // publishDir "${OUTDIR}/contamination", mode: 'copy', overwrite: 'true', pattern: '*.selfSM'
```

Comment:
Are these intended to be published? Or not published, just part of the QC outputs. If intended to be published in the long run, I would uncomment before merge. If unsure, we could discuss in the RD BNF meeting tomorrow.

Reply:
Dont know really, depends what we would like to do.

Reply:
Yes, maybe we can discuss this tomorrow before merge

Reply:
(Or after merge, this could come later)